### ISSUE TITLE
Add an expression for reading contents of files

### ISSUE THREAD

**Issue by loreanvictor** [loreanvictor/tmplr#4]
This can be used to _include_ some files within each other, better yet to conditionally _include_ sections and what not:

```yml
# .tmplr.yml
steps:
  - if: some_condition
    read: header
    from file: ./_some_case.md
    else:
      read: header
      from file: ./_some_other_case.md

  - update: README.md
```
```markdown
<!-- README.md -->

{{ tmplr.header }}

# Some other stuff

Ha ha ho ho
```

**Comment by loreanvictor** [comment]
Similar to [the proposed `write` command](#23), this command could also support reading special values from JSON / YAML files.

```yaml
read: some_variable
from file: conf.json
at: /path/to/key
```

**Comment by loreanvictor** [comment]
resolved via 8ce8644
